DESIGN

```diff
@@ -170,6 +170,14 @@ ProvisionedThroughputExceededException code):
 * PUT(key, len, data):
 	Associate the provided (binary) data with the provided (string) key.
 	Return 0 for success or 1 for failure.
+* ICAS(key, len, data, len2, data2):
```

Comment:
Does this stand for "Idempotent Compare And Swap"?  It might be helpful to spell that out in the blurb.